linux/jdk/suse/build.gradle

```diff
@@ -3,9 +3,9 @@ plugins {
 }
```

Comment:
seems to be some common functionality in these scripts, can we extract?

Reply:
it is quite interesting. 
most of the config in gradle already defined in the root project (linux/build.gradle), but then in different distro they are defined again. guess it is for overwritten purpose when it was tested. 
these lines of code should be able to remove

Reply:
i am thinking to leave the "code improvement" into another issue, so we can get review done for the current change first